app/src/main/java/com/example/parkingsystemkotlin/mvp/view/base/FragmentView.kt

```diff
@@ -8,4 +9,8 @@ open class FragmentView(fragment: Fragment) {
 
     val fragment: Fragment?
         get() = fragmentRef.get()
+
+    val dialogFragment
+        get() = (fragment as DialogFragment?)
```

Comment:
be careful with this. It is not fine because if are using another type of fragmented in the future and you use this variable you will get a crash. Please put this logic only when you need it, not here

Reply:
I chose this solution because in this case I am only going to work with dialogFragments, it is true that it is not extensible, as the first option I had made a solution like you give me, but then I realized that for the only two fragments it is the same case

Reply:
yes, you are using only DialogFragments but will create this code to be extensible to keep the good practices ;)

Reply:
Perfect, done